labs/Demos/Cryptol/OneTimePad.md

```diff
@@ -269,9 +271,10 @@ reasonably have been "GOODBYE". Then Eve can exploit Cryptol to
 deduce the pre-shared key Bob just used:
 
 ```shell
-labs::Demos::Cryptol::OneTimePad> :sat \psk -> (encrypt psk pt2) == ct2
-(\psk -> (encrypt (psk : String 7) pt2) == ct2) "ZUGESAG" = True
-(Total Elapsed Time: 0.042s, using Z3)
+labs::Demos::Cryptol::OneTimePad> :sat \psk2 -> (encrypt`{7} psk2 pt2) == ct2
+Satisfiable
```

Comment:
I’m confused why this line prints the word Satisfiable but line 294 doesn’t.

Reply:
That was an error on my part.  When running a Cryptol module in batch mode, but not when running the interpreter interactively, `:sat` commands show `Testing...` and `Satisfiable`.